bin/l2l-fun-ce-gaussmix.py

```diff
@@ -56,6 +57,14 @@ def main():
 
     # Get the trajectory from the environment
     traj = env.trajectory
+    # Set JUBE params
+    traj.f_add_parameter_group("JUBE_params", "Contains JUBE parameters")
```

Comment:
Do you need this or are the parameters missing?

Reply:
This is needed to have a specific parameter group called JUBE_params. There you can add all the related parameters to the JUBE_runner